app/models/article.rb

```diff
@@ -1,4 +1,4 @@
 class Article < ApplicationRecord
   belongs_to :user
-  validates :title, :body, presence: true
+  validates :title, :body, :user_id, presence: true
```

Comment:
In this case it is unnecessary because `belongs_to :user` relation already provider validation for this 
```suggestion
  validates :title, :body, presence: true
```